.cirrus.yml

```
@@ -0,0 +1,22 @@
+aliases:
+  - &bundle_cache
```

I think the syntax should be like this:

```suggestion
  bundle_cache: &bundle_cache |
```

---

Why (for what) `|`?

---

Still no…

---

`|` for multiline strings, as I know.

Aliases are correct, and parsed (in Ruby) correctly.

---

You can check YAML file here, for example: https://yaml-online-parser.appspot.com/